### ISSUE TITLE
Missing where on firestore collections

### ISSUE THREAD

**Issue by ocombe** [soumak77/firebase-mock#41]
I get an error about `prs.where is not a function` when I try the following code with firebase mock:

```ts
const store = new MockFirestore();
const prs = store.collection('prs');
prs.where('repository', '==', repository.id)
   .where('state', '==', 'open')
   .get()
```

This is valid firestore code (it works without the mock).

For now I've patched it with

```ts
prs.where = () => prs;
```

**Comment by soumak77** [comment]
@ocombe I've published version 2.0 which has support for `where`.  Currently it only supports the `==` operator.

**Comment by soumak77** [comment]
@amirhmk `==` is still the only supported operator.  I'll gladly take any PRs which help implement the other operators.
